File: src/main/java/com/spring/spring_init/verify/dto/request/VerifyPasswordResetRequest.java
package com.spring.spring_init.verify.dto.request;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class VerifyPasswordResetRequest {

    @NotNull(message = "uid 필수 값 입니다.")
    private String uid;

    @NotNull(message = "token 필수 값 입니다.")
    private String token;
}

File: src/main/java/com/spring/spring_init/verify/dto/request/VerifyPasswordResetRequest.kt
package com.spring.spring_init.verify.dto.request

import jakarta.validation.constraints.NotNull

data class VerifyPasswordResetRequest(
    @field:NotNull(message = "uid 필수 값 입니다.")
    val uid: String,

    @field:NotNull(message = "token 필수 값 입니다.")
    val token: String
)
